src/main/java/Duke.java

```diff
@@ -1,10 +1,74 @@
+import java.util.Scanner;
+
 public class Duke {
+    static String horizontal = " __________________________________________________\n";
+    static String introduction = horizontal + "   Hello! I'm Duke\n" + "   What can I do for you?\n" + horizontal;
+    static String farewell = horizontal + "   Bye. Hope to see you again soon!\n" + horizontal;
```

Comment:
These strings are constant throughout your application, so they should be named in all caps.

i.e. HORIZONTAL, INTRODUCTION, FAREWELL

I like what you have done with this though, easier to change your messages if required

Reply:
You can maybe even consider moving all of this into a print welcome message method?

Reply:
Also if you're going to make them constants, I think should change them to static final as well.